test/fixtures/petStore.no_host.json

```diff
@@ -0,0 +1,1051 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is a sample server Petstore server.  You can find out more about Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For this sample, you can use the api key `special-key` to test the authorization filters.",
```

Comment:
I am strongly against boilerplate test fixtures - how can I tell what's the difference between `petStore.no_host.json` and `petStore.json`?

Ideally, each test should use a minimal Swagger document describing exactly the endpoints that are needed for the test.

The second best option is to load `petStore.json` and make any changes needed in the test code, i.e. delete the top-level `host` property.